matlab/gtsam_examples/VisualizeMEstimators.m

```diff
@@ -0,0 +1,172 @@
+import gtsam.*
```

Comment:
Please add documentation and comments? Take a look at LocalizationExample.m to see the documentation standards. In addition, could we scale all figures the same way so the models are directly comparable?